### PR TITLE
Added new courses

### PR DESCRIPTION
**Pull request by mkghosh**
@ericdouglas I have added new courses please review and let me know if those can be added.

### CODE REVIEW

File: extras/courses.md [hunk 1]
```diff
@@ -129,9 +129,11 @@ Courses | Duration | Effort
 :-- | :--: | :--:
 [Applied Cryptography](https://www.udacity.com/course/applied-cryptography--cs387)| 8 weeks | 6 hours/week
 [Cryptography](https://www.coursera.org/learn/cryptography/)| 7 weeks | 4-7 hours/week
-
-
+[Cryptography I](https://www.coursera.org/learn/crypto/)| 7 weeks | 4-7 hours/week
```
Comment (ericdouglas) [inline extras/courses.md]: We already have this course in the main curriculum. Could you remove this one (Cryptography I)?

**Comment by mkghosh** [comment]
@ericdouglas Sure

**Comment by mkghosh** [comment]
@ericdouglas I have removed Cryptography I

**Comment by ericdouglas** [comment]
@mkghosh great! Just one more thing: you need to pull [this](https://github.com/open-source-society/computer-science/commit/0d0f5db39f67d0e7438f14a51a3c8078a4d10507) commit, to fix the header of the Mathematical Thinking section.

**Comment by mkghosh** [comment]
@ericdouglas okay

**Comment by mkghosh** [comment]
@ericdouglas I have pulled and updated my local branch thank you.

**Comment by ericdouglas** [comment]
@mkghosh thank you so much! :pray: :smile:

**Pull request merged**